### ISSUE TITLE
Renal interstitial non-monoclonal immune-type deposits (i.e., IgG4)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5058]
Definition:Expansion of the renal interstitium by fibrilar, non-amyloidogeneic, non-monoclonal deposits (i.e., IgG4 disaease)

**Comment by pnrobinson** [comment]
@avizrosenberg
@yongqunh  
@beckysteck

I am wondering if the observed phenotypic feature is better described as

Elevated IgG4-positive plasma cell count

see e.g., https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5325071/

**Comment by pnrobinson** [comment]
@avirosenberg
I see now that we already have this term
Renal interstitial immunoglobulin deposits
https://hpo.jax.org/app/browse/term/HP:0032616

Is this synonymous with the "grandparent" term we discussed yesterday ?

(grandparent) Renal interstitial non-monoclonal immune-type deposits
--(child) Renal interstitial non-monoclonal IgG deposits
----(grandchild) Renal interstitial non-monoclonal IgG4 deposits

**Comment by pnrobinson** [comment]
Adding term as
label: Renal interstitial IgG4+ plasma cell infiltration
Def: An increased number of IgG4+ plasma cells in the interstitial space of the kidney.
PMID: 29359118  PMID: 21719792 
Comment: IgG4-related tubulointerstitial nephritis is characterized by an increased number of IgG4+ plasma cells per high-power field on renal biopsy in addition to either elevated serum IgG4 or evidence of extra renal IgG4-related disease. 


@avirosenberg please reopen if this is incorrect.
